python_course_1604/class_01/tests/test_04.py

```diff
@@ -12,6 +12,7 @@
 
 
 @skip_if_exercize_not_started(exercize)
+@unittest.skip('It brokes coverage.')
```

Comment:
1. I don't understand the syntax of your unittest. Does the first @ mean context? Why a space between @unittest. and skipIf( and why the second unittest below it? Why not just @unittest.skipIf(socket.gethostname() == 'omalleypark-1')?
2. Is the full host name just omalleypark-1 or omalleypark-1.ir.intel.com?
3. What does import socket do?